Source/Renderer/VertexArray.js

```diff
@@ -116,15 +126,119 @@ define([
     }
 
     /**
+     * Creates a vertex array, which defines the attributes making up a vertex, and contains an optional index buffer
+     * to select vertices for rendering.  Attributes are defined using object literals as shown in Example 1 below.
+     *
+     * @param {Object} options Object with the following properties:
+     * @param {Context} options.context The context in which the VertexArray gets created.
+     * @param {Object[]} options.attributes An array of attributes.
+     * @param {IndexBuffer} [options.indexBuffer] An optional index buffer.
+     *
+     * @returns {VertexArray} The vertex array, ready for use with drawing.
+     *
+     * @exception {DeveloperError} Attribute must have a <code>vertexBuffer</code>.
+     * @exception {DeveloperError} Attribute must have a <code>componentsPerAttribute</code>.
+     * @exception {DeveloperError} Attribute must have a valid <code>componentDatatype</code> or not specify it.
+     * @exception {DeveloperError} Attribute must have a <code>strideInBytes</code> less than or equal to 255 or not specify it.
+     * @exception {DeveloperError} Index n is used by more than one attribute.
+     *
+     * @see VertexArray.fromGeometry
```

Comment:
Remove these `@see`s too since they are going to be out of date shortly and this is just internal doc since the type is `@private`.